### DISCUSSION TITLE
Why does render a Jupyter Notebook (.ipynb) creates the _freeze directory

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#9790] [Q&A] [answered]
### Description

Given any Jupyter Notebook (we will called our file `index.ipynb`), when I run `quarto render index.ipynb --execute`, Quarto will

1. update the content of `index.ipynb` with the output of the code cells
2. create and populate the `_freeze` directory

**Why does Quarto creates the `_freeze` directory?**

From my experimentation, the content of the `_freeze` directory is ignored for Jupyter Notebooks.

**Comment by mcanouil** [comment, marked as answer]
I cannot reproduce.

Did you set `freeze` somewhere in your project?


````qmd
---
title: "Quarto Playground"
format: html
---

This is a playground for Quarto.

![An image](https://placehold.co/600x400.png)


```{python}
1 + 1
```
````


````sh
quarto convert index.qmd
quarto render index.ipynb --execute
````

**Reply by rgaiacs** [reply]
@mcanouil Thanks for the answer.

> Did you set freeze somewhere in your project?

Yes. I forgot that `_quarto.yml` had

```
execute:
    freeze: true
```
